optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -574,6 +574,121 @@ the stream.
 ====
 
 
+[[constraintStreamsTesting]]
+== Testing a constraint stream
+
+Constraint streams come with a unit testing harness, called "Constraint Verifier".
+Consider the following constraint stream:
+
+[source,java,options="nowrap"]
+----
+    protected Constraint horizontalConflict(ConstraintFactory factory) {
+        return factory
+                .fromUniquePair(Queen.class, equal(Queen::getRowIndex))
+                .penalize("Horizontal conflict", SimpleScore.ONE);
+    }
+----
+
+This is how a simple unit test for this constraint looks, using the Constraint Verifier API:
```

Comment:
"this is how" = too indirect. Start with the important keywords.
Try something like "A simple unit test for this constraint looks like this:"

Reply:
The following example uses the Constraint Verifier API to create a simple unit test for the preceding constrain stream: